public/data/Mustafiz04.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Mustafiz Kaifee",
+  "type": "personal",
+  "bio": "Hi there! I am backend developer at Freecharge.",
+  "avatar": "https://github.com/Mustafiz04.png",
+  "links": [
+    {
+      "name": "Take a look at my GitHub",
+      "url": "https://github.com/Mustafiz04",
+      "icon": "github"
+    },
+    {
+      "name": "See my awesome tweets on Twitter",
+      "url": "https://twitter.com/mustafiz_kaifee",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/mustafizkaifee/",
```

```suggestion
      "url": "https://www.linkedin.com/in/mustafizkaifee",
```

The slash isn't required in the end of the link, the link works even without the slash, the slash only comes when you copy it the url from the taskbar in google or in other browsers, but it's not required.